lib/game.rb

```diff
@@ -0,0 +1,106 @@
+class Game
+
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [
+   [0, 1, 2],
+   [0, 3, 6],
+   [0, 4, 8],
+   [1, 4, 7],
+   [2, 4, 6],
+   [2, 5, 8],
+   [3, 4, 5],
+   [6, 7, 8]
+  ]
+
+  def initialize(player_1 = Player::Human.new("X"), player_2 = Player::Human.new("O"), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    @board.turn_count.even? ? @player_1 : @player_2
+  end
+
+  def over?
+    draw? || won? ? true : false
+  end
+
+  def won?
+    WIN_COMBINATIONS.each do |combo|
+      if @board.cells[combo[0]] == "X" && @board.cells[combo[1]] == "X" && @board.cells[combo[2]] == "X"
+        return player_1
+      elsif @board.cells[combo[0]] == "O" && @board.cells[combo[1]] == "O" && @board.cells[combo[2]] == "O"
+        return player_2
+      end
+    end
+    return false
+  end
+
+  def draw?
+    @board.full? && !won? ? true : false
+  end
+
+  def winner
+    if winner = won?
+      winner.token
+    else
+      nil
+    end
+  end
+
+  def turn
+    p = current_player.move(@board)
```

Comment:
`p` is a method in ruby equivalent to `puts <variable>.inspect`. You should avoid using this for a local variable name as it may be confusing to other programmers looking at your code.